_assets/js/netlify/topics/generateChildElements.js

```diff
@@ -0,0 +1,30 @@
+const notAllowed = ['em', 'strong', 'u', 'i', 'b', 'ul', 'blockquote', 'table', 'thead', 'td', 'tr','tfooter', 'tbody', 'br', 'input'];
+
+const generateChildElements = (html) =>
+  html.map((item) => {
+    if (item.children && item.children.length > 0) {
```

Comment:
are we allowed to use optional chaining?

```suggestion
    if (item.children?.length > 0) {
```

Reply:
Totally allowed! I'm not used to using this operator.